### PR TITLE
move some ADRs to dev-docs/

### LINKED ISSUE #492: Remove adrs & architecture folders

### PR DESCRIPTION
**Pull request by rkuhn**
closes #492

### CODE REVIEW

File: rust/actyx/Rust crate structure.md
Comment (jmg-duarte) [inline rust/actyx/Rust crate structure.md:1]: Isn't this content already covered in the folder's README?

**Review by jmg-duarte — APPROVED** [review]
Approving because the remark is not that important, but asides from it, LGTM.

File: rust/actyx/README.md [hunk 1]
```diff
@@ -17,16 +17,21 @@ test a specific subproject you can you the `-p` option:
 
 ## Map of our components
 
-All data types used across serialisation boundaries should be moved to the `SDK`, which removes the need for `*-formats` crates. Local storage is managed by the `ipfs-sqlite-block-store` crate. Then, we need the following:
+All data types used across serialisation boundaries should be moved to the `SDK`, which removes the need for `*-formats` crates.
+Local storage is managed by the [`ipfs-sqlite-block-store`](https://github.com/Actyx/ipfs-sqlite-block-store) crate.
```
Comment (Kelerchian) [inline rust/actyx/README.md:21]: Just an appreciation that this section makes Actyx's internals much clearer

**Review by Kelerchian — APPROVED** [review]
Just one bit of a detail:

The file `adrs/2021-04-12 of unit and integration tests.md` seems rather informative to justify the existence of the tests. Is it removed because some parts of it are outdated?

**Comment by rkuhn** [comment]
Regarding the testing ADR: yes, most of it covers tests that we no longer have. We could revive an excerpt of this document later.

**Pull request merged**